plugins/inputs/ceph/README.md

```diff
@@ -205,180 +229,216 @@ All admin measurements will have the following tags:
 
 - ceph_usage
   - fields:
-    - total_bytes (float)
-    - total_used_bytes (float)
+    - num_osd (float)
+    - num_per_pool_omap_osds (float)
+    - num_per_pool_osds (float)
+    - total_avail (float, exists only in ceph <0.84)
     - total_avail_bytes (float)
+    - total_bytes (float)
     - total_space (float, exists only in ceph <0.84)
     - total_used (float, exists only in ceph <0.84)
-    - total_avail (float, exists only in ceph <0.84)
+    - total_used_bytes (float)
+    - total_used_raw_bytes (float)
+    - total_used_raw_ratio (float)
+
+- ceph_deviceclass_usage
+  - tags:
+    - class
+  - fields:
+    - total_avail_bytes (float)
+    - total_bytes (float)
+    - total_used_bytes (float)
+    - total_used_raw_bytes (float)
+    - total_used_raw_ratio (float)
 
 - ceph_pool_usage
   - tags:
     - name
   - fields:
-    - kb_used (float)
     - bytes_used (float)
+    - kb_used (float)
```

Comment:
This is the same but only a factor 1000 (or 1024) different?

But yeah, I know, both fields existed before as well, so we need to keep both of them..